meson.build

```diff
@@ -14,7 +14,6 @@ project(
 add_project_arguments(
 	[
 		'-DWLR_USE_UNSTABLE',
-
```

Comment:
Kind of a nit but I think we should keep this newline in, it's present on sway and wlroots to separate the WLR args